spot-client/src/common/logger/post-to-endpoint.js

```diff
@@ -1,18 +1,60 @@
+
+/**
+ * The structure holds the meta data about the log requests waiting in the queue
+ * to be sent.
+ */
+class PostLogsRequest {
```

Comment:
This class was a really good idea. Really good way to mentally keep together some variables.